sources/Common/include/Dico.h

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright (c) 2020, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
+#pragma once
```

Comment:
Cannot we reuse dynawo classes for this?

Reply:
I'm not sure of the behaviour of the singleton IoDicos if we use it directly